tests/test_templates.py

```diff
@@ -0,0 +1,59 @@
+import unittest
+import os
+import mock
+from trigger.utils.templates import *
+from trigger.conf import settings
+from contextlib import contextmanager
+from StringIO import StringIO
+import cStringIO
+
+
+try:
+    import textfsm
+except ImportError:
+    print("""
+    Woops, looks like you're missing the textfsm library.
+
+    Try installing it like this::
+
+        >>> pip install gtextfsm
+    """)
+
+
+cli_data = """*02:00:42.743 UTC Sat Feb 20 2016"""
+
+text_fsm_data = """Value TIME (\d+:\d+:\d+\.\d+)
+Value TIMEZONE (\w+)
+Value DAYWEEK (\w+)
+Value MONTH (\w+)
+Value DAY (\d+)
+Value YEAR (\d+)
+
+Start
+  ^[\*]?${TIME}\s${TIMEZONE}\s${DAYWEEK}\s${MONTH}\s${DAY}\s${YEAR} -> Record
+"""
+
+class CheckTemplates(unittest.TestCase):
+    """Test structured CLI object data."""
+
+    def setUp(self):
+        data = cStringIO.StringIO(text_fsm_data)
+        self.re_table = textfsm.TextFSM(data)
+        self.assertIsInstance(self.re_table, textfsm.textfsm.TextFSM)
```

Comment:
`assertIsInstance()`is Python 2.7+. We still need to maintain 2.6 support for now, so can you please replace this with something like:

```python
        self.assertTrue(isinstance(self.re_table, textfsm.textfsm.TextFSM))
```